api/e2e/test/08_deploy_router_with_traffic_rules_test.go

```diff
@@ -162,6 +162,13 @@ func TestDeployRouterWithTrafficRules(t *testing.T) {
                           },
                           "is_default": false,
                           "route": "control"
+                        },
```

Comment:
I'm a bit confused by this. Shouldn't the default rule be activated for this case which only calls the control route?

Reply:
Good catch, this is caused by incorrect assertion of the test, I've fixed it.